packages/uhk-web/src/app/components/agent/about/contributor-badge/contributor-badge.component.ts

```diff
@@ -0,0 +1,23 @@
+import { Component, Input } from '@angular/core';
+import { UHKContributor } from '../../../../models/uhk-contributor';
+
+@Component({
+    selector: 'contributor-badge',
+    templateUrl: './contributor-badge.component.html',
+    styleUrls: ['./contributor-badge.component.scss']
+})
+export class ContributorBadgeComponent {
+    @Input() contributor: UHKContributor;
+
+    get name(): string {
```

Comment:
I just would like to ask what is the benefit of using property getters instead of `{{ object.property }}` in the template?

Reply:
It's just clearer. GitHub API property names don't communicate their purpose clearly, plus they break lowerCamelCase policy.

Reply:
ok, thx